ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/Main.scala

```diff
@@ -113,15 +112,17 @@ object Main extends StrictLogging {
     val name = "http-json-perf"
     val terminationTimeout: FiniteDuration = 30.seconds
 
-    implicit val asys: ActorSystem = ActorSystem(name)
-    implicit val mat: Materializer = Materializer(asys)
+    val asys: akka.actor.ActorSystem = akka.actor.ActorSystem(name)
```

Comment:
I remove the implicit for the akka ActorSystem, so we see clearly it is use only once.